frontend/src/app/commonComponents/TestResultGuidance/FluResultGuidance.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
```

Comment:
⚠️ most of the content in this file was taken from `MultiplexResultsGuidance.tsx` which is now deleted